protocol-configuration/mainnetAddresses.ts

```diff
@@ -1981,6 +1981,11 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0xB80B3dc4F8B30589477b2bA0e4EF2b8224bDf0a5',
     category: AddressCategory.PCV
   },
+  compoundUnderlyingPCVDepost: {
```

Comment:
to be consistent, can you name it `compoundFeiPCVDeposit` ? and rename `compoundPCVDepositWrapper` to `compoundFeiPCVDepositWrapper`

Reply:
yep